triggers.cwt

```diff
@@ -3531,3 +3548,10 @@ alias[trigger:heavy_troops] = {
     who = <special_troop_type>
     value = float
 }
+
+# TODO: Docs
+## scope = character
+### 
+alias[trigger:defender] = {
```

Comment:
Like with attacker I'm not sure that's a general trigger and not something local to wherever it's used.

Reply:
It's used in decisions and events.